### PR TITLE
[Fix #14] Create PROJECT_GOVERNANCE.md

### LINKED ISSUE #14: Governance of maintained projects
@hlship raises some good questions about how to vizdeps should be managed, and also more generally how CLJ Commons projects should be managed: https://github.com/clj-commons/vizdeps/issues/5.

* What is the PR review protocol? For example, is one review enough, or should changes be reviewed by two committers?
* Is there a CLA?
* Who determines when to perform a release?
* Who is responsible for the actual release?
* Where do we make post-release announcements?

I also raised some questions here on the governance of adopted projects: https://clojureverse.org/t/clj-commons-first-project-resurrecting-unmaintained-projects/3164

* Do we want to set some entry requirements for people that are willing to be responsible for maintenance?
* Should we set some exit criteria for projects that we adopt but then become superceded or no longer used? Where do those projects go?
* I think it would be quite useful to have some number of people (more than one?) who commit to maintaining this new project under the clj-commons banner. We don’t want the new home of the project to become unmaintained as well.
* What are the minimum requirements for maintenance of a project? Some ideas:
  * Passing continuous integration
  * Responsive to issues and pull requests
  * Documentation
  * Regularly creating releases
  * Staying up to date with new Clojure and JVM releases

### PR DESCRIPTION
**Pull request by slipset**
I've tried to distill some of the questions asked in #14. Please feel free to add/comment

### CODE REVIEW

File: PROJECT_GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  # Transfer vs forking
+ When a project is adopted to clj-commons, a transfer of ownership should be preferred over forking.
+ There are several reasons for this:
+
+  1) The adoption is somewhat formalized and friendly, the original authors need to give their blessing for
+ this to happen.
+
+  2) All github metainfo, stars, issues, PRs, etc are transferred in a transfer, but lost in a fork
+
+  We should resort to forking only if the original maintainers remain unresponsive to our inquiries.
```
Comment (MalloZup) [inline PROJECT_GOVERNANCE.md]: we could add also a tip here like consider to contact the author also via e-mail

Comment (MalloZup) [reply PROJECT_GOVERNANCE.md]: but not really mandatory

Comment (seancorfield) [reply PROJECT_GOVERNANCE.md]: I would just assume we should try whatever contact avenues we have. I don't think the wording needs to be changed to include every possible avenue.

**Comment by seancorfield** [comment]
@MalloZup I don't think we should be overly prescriptive about how maintainers run the projects, re: no breaking changes. It's a general principle of Clojure as a whole but it's up to individual maintainers whether they follow that.

File: PROJECT_GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Each project under clj-commons need to have at least a maintainer, and the names of the maintainers should be listed
+ in the `.github/CODEOWNERS` file. The organization as such do not pose any 
+ restrictions on how the individual projects are run, other than what is stated in the 
+ [meta/README](https://github.com/clj-commons/meta/README.md).
+
+  This means that PR review protocol, release cycles, and post-release annoncements are up to the 
+ maintainer of the project, even though we encourage announcing new releases to the main Google Group
+ and the #announcement channel in the Clojurians slack. Also, coding style is up to the maintainers of
```
Comment (rborer) [inline PROJECT_GOVERNANCE.md]: Would probably be interesting to add links to the main Google Group and Clojurians slack. Probably not everyone is aware of those 2 communities (I'm a long time Clojure user and discovered Clojurians slack a few months ago only).

File: PROJECT_GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ the project, but clj-commons recommend following [The Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide)
+
+  #Continious Integration
+ As clj-commons as an org is most familiar with [CircleCI](https://circleci.com), we recommend using CircleCI
+ as a projects CI tool.
```
Comment (rborer) [inline PROJECT_GOVERNANCE.md]: What about mentioning that CircleCI is configured at the org level already? Meaning that simply adding the CircleCI config files is sufficient to enable CI.

Comment (slipset) [reply PROJECT_GOVERNANCE.md]: @rborer: good points here and above, PR welcome :)

Comment (rborer) [reply PROJECT_GOVERNANCE.md]: @slipset see #50 :-)

**Pull request merged**